packages/modules/web_themes/colors/source/src/components/powerGraph/PowerGraph.vue

```diff
@@ -25,8 +25,18 @@
 				</span>
 			</div>
 		</template>
-
-		<figure id="powergraph" class="p-0 m-0" @click="changeStackOrder">
+		<div
+			v-if="graphData.data.length == 0"
+			class="d-flex justify-content-center waitsign rounded"
+		>
+			<span class="fa-solid fa-xl fa-spinner fa-spin"></span>
```

Comment:
Bringt ein leeres span was?

Reply:
Yep. Die fa-*-Klassen sorgen für die Anzeige von Icons. In diesem Fall ein rotierendes Icon, während der Graph neu berechnet wird.